hnn_core/tests/test_network.py

```diff
@@ -115,7 +175,7 @@ def test_network():
         assert len(net.gid_ranges[dn]) == n_drive_cells
 
     # Check drive dict structure for each external drive
-    for drive in net.external_drives.values():
+    for drive_idx, drive in enumerate(net.external_drives.values()):
```

Comment:
should these tests go to a separate function `test_add_drives` ? Currently our test functions are a little monolithic. Separating them out a bit so they check individual components rather than "combinations of components" will be helpful

Reply:
I've now separated the `test_network()` function in `test_network.py` into two parts: `tests_network_drives()` and `tests_network_connectivity()`.